podcasts/End of Year/StoriesModel.swift

```diff
@@ -122,7 +122,8 @@ class StoriesModel: ObservableObject {
     }
 
     func loadingStart() {
-        loadingCancellable = publisher.autoconnect().sink(receiveValue: { _ in
+        loadingCancellable = publisher.autoconnect().sink(receiveValue: { [weak self] _ in
+            guard let self else { return }
```

Comment:
This was my first guess as well looking at that Slack thread but I couldn't reproduce with multiple `loadingStart` calls.